Add a description about "inject-ca-from-secret" to the cainjector ref…

bells17 opened the pull request:
**What this PR does / why we need it:**

Hi,
I added a description about "inject-ca-from-secret" to the cainjector reference page because a description is not found.
Please review if you like.

munnerz commented on docs/reference/cainjector.rst:
```diff
@@ -10,3 +10,6 @@ APIService resources annotated with:
   Injects the cluster CA.
 * ``cert-manager.io/inject-ca-from: <NAMESPACE>/<CERTIFICATE>``  
   Injects the CA from the specified :doc:`certificate </reference/certificates>`.
+* ``cert-manager.io/inject-ca-from-secret: <NAMESPACE>/<CERTIFICATE>``  
```
maybe change `CERTIFICATE` to `SECRET` here? 😄

bells17 replied:
Year, thank you 😅 
I’ll fix now.

munnerz commented on docs/reference/cainjector.rst:
```diff
@@ -10,3 +10,6 @@ APIService resources annotated with:
   Injects the CA from the specified :doc:`certificate </reference/certificates>`.
+* ``cert-manager.io/inject-ca-from-secret: <NAMESPACE>/<CERTIFICATE>``  
+  Injects the CA from the specified `secret <https://kubernetes.io/docs/concepts/configuration/secret/>`_.  
+  the secret is needed the ``cert-manager.io/allow-direct-injection: "true"`` annotation
```
nit: capitalise `The`
also, can you rephrase this to
```
The secret resource must have the ``cert-manager.io/allow-direct-injection: "true"`` annotation to allow it to be injected into other resources.
```

bells17 commented:
Thank you for your quickly response 😃 
And sorry, I fixed from Github Edit File. But it wasn’t able to signed commit...
I’ll fix that. please wait a minutes.

munnerz commented:
`git rebase --signoff master` should do it 😄

bells17 commented:
@munnerz
I didn't know it can use the `--signoff` option in `git rebase`.
I'll try it next time 😃 (I have already modified this time...).
And I fixed issues that are commented on.  Please review it when you have a time 🙏

bells17 commented:
@munnerz Hi, Could you review this one again?
And if I need some add fixings, I would be grateful if you teach me it.

meyskens commented:
Documentation has moved to https://github.com/cert-manager/website
So going to close this PR here

/close
